### PR TITLE
Module 1 : Hello Microverse project #1

### PR DESCRIPTION
**Pull request by EzekielUtshudi**
## Hello world project setup
We added : 
- we configured HTML, CSS, and Github actions lints
- We added HTML, CSS and README files

### CODE REVIEW

**Review by ReshyResh — APPROVED** [review]
# Hi @EzekielUtshudi , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it!
#### Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by EzekielUtshudi** [comment]
Hi ,

My code all passed the review but i receive a second email that says Error
but BELOW is the screenshot of my code,
[image: image.png]

Yours Faithful.
Ezekiel Utshudi

File: README.md [hunk 1]
```diff
@@ -1,2 +1,76 @@
...
+
+- GitHub: [@ezekielutshudi](https://github.com/EzekielUtshudi)
+- Twitter: [@ezekielutshudi](https://twitter.com/UtshudiEzekiel)
+- LinkedIn: [@ezekielutshudi](https://www.linkedin.com/in/ezekiel-utshudi-195782162/)
+
+<!--👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)-->
```
Comment (Whoistolu) [inline README.md:56]: - [ ] [OPTIONAL] Kindly remove all unused line of code in order to have a clean code-base

File: README.md [hunk 2]
```diff
@@ -1,2 +1,76 @@
...
+-cd \<project-name>
+
+### Install
+- nom Install
+
+<!--### Usage -->
+
+<!--### Run tests-->
+
+<!--### Deployment-->
```
Comment (Whoistolu) [inline README.md:40]: - [ ] [OPTIONAL] Kindly remove all unused line of code in order to have a clean code-base

**Review by Whoistolu — APPROVED** [review]
### Hi @EzekielUtshudi 

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

- [ ] [OPTIONAL] Kindly ensure the first alphabet of your commit messages is a capital letter

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by EzekielUtshudi** [comment]
Awesome, thank you so much!

**Review by CollinsTatang — APPROVED** [review]
Hi @EzekielUtshudi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (EzekielUtshudi) [reply README.md:56]: [OPTIONAL] Please kindly remove all the code that is commented in your README.md file to keep it clean.

**Pull request merged**